jupyterlab_latex/build.py

```diff
@@ -116,6 +118,8 @@ def build_tex_cmd_sequence(self, tex_base_name, run_bibtex=False):
                 full_latex_sequence,
                 ]
 
+        command_sequence = command_sequence*c.run_times
```

Comment:
I don't know much about how `hyperref` and `tableofcontents` work.Is it overkill to do this in the case of where we use `bibtex`? An alternative could be `pdflatex bibtex pdflatex pdflatex` for when `bibtex` is enabled, and `[pdflatex]*c.run_times` when it is not being used.

Of course, if that doesn't work, feel free to educate me.

Reply:
So maybe what I had here before was better? Of course it'll work, but in that case `run_times` would have no effect, if `run_bibtex=True`. 

I'm not really familiar with `bibtex` either, so I don't know if it might ever be necessary to run multiple times, but if so, I think it should also be using `run_times`.
Maybe something like 

        command_sequence = [full_latex_sequence]

        if run_bibtex:
            command_sequence += [full_bibtex_sequence]*c.run_times

        command_sequence += [full_latex_sequence]*c.run_times

would work, but that would mean `latex` would run twice, if `run_bibtex=False` and `run_times=1`.

Reply:
I think I got it. Do you know if it might ever be necessary to run `bibtex` multiple times? If yes, I would to this:
        command_sequence = []

        if run_bibtex:
            command_sequence = [full_latex_sequence] + [full_bibtex_sequence]*c.run_times + [full_latex_sequence]

        command_sequence += [full_latex_sequence]*c.run_times

or if not:
        command_sequence = []

        if run_bibtex:
            command_sequence = [full_latex_sequence, full_bibtex_sequence, full_latex_sequence]

        command_sequence += [full_latex_sequence]*c.run_times

Reply:
Alright, almost missed your last comment. I made the changes and will add a commit now.

Reply:
I don't think it should be necessary to run `bibtex` multiple times, at least not as I understand it.

If you want to not use an `else` clause, how about we default `run_times` to `2`, and then remove one of those `full_latex_sequences`? So
```python
        command_sequence = []

        if run_bibtex:
            command_sequence = [full_latex_sequence, full_bibtex_sequence]

        command_sequence += [full_latex_sequence]*c.run_times
```

Reply:
I'm sorry, I shouldn't have just removed my comment. Anyways I'm fine with what you said before. Have created another commit now.